### PR TITLE
AndroidX Support - Requires corresponding Uno PR

### PR DESCRIPTION
**Pull request by AndrewEQ**
Migrated support libraries to AndroidX
https://developer.android.com/jetpack/androidx/migrate

Requires this PR first: https://github.com/fuse-open/uno/pull/254

### CODE REVIEW

**Comment by mortend** [comment]
Now you can try `npm install @fuse-open/uno@canary` to get Android X support in Uno.

**Comment by AndrewEQ** [comment]
How can we get this to rebuild?

**Comment by mortend** [comment]
You need `npm install -D @fuse-open/uno@canary` to get androidx (and preferably rebase on master).

**Comment by AndrewEQ** [comment]
oh, I meant, how can we get this PR to rebuild to pass its checks based on Uno with AndroidX

**Comment by mortend** [comment]
The canary version of Uno includes the AndroidX code, so if you install that version as part of this PR the checks should pass.

**Comment by AndrewEQ** [comment]
I'm confused as to where to put this in travis-ci and appveyor?

**Comment by mortend** [comment]
When you install it in `package.json` on this repo it'll be automatically picked up by AppVeyor and Travis.

**Comment by AndrewEQ** [comment]
Oh I see now, wow, that's pretty cool 😄

File: Source/Fuse.GeoLocation/Android/AndroidLocationProvider.uno [hunk 1]
```diff
@@ -8,7 +8,8 @@ using Fuse.GeoLocation.Android;
 
 namespace Fuse.GeoLocation
 {
-	[ForeignInclude(Language.Java, "android.support.v4.content.ContextCompat", "android.content.pm.PackageManager", "android.Manifest", "android.location.LocationManager", "android.location.Location", "android.provider.Settings", "android.util.Log", "java.util.List", "fuse.geolocation.UpdateListener", "android.os.Looper", "android.content.Context", "com.uno.StringArray")]
+	[ForeignInclude(Language.Java, "androidx.core.content.ContextCompat", "android.content.pm.PackageManager", "android.Manifest", "android.location.LocationManager", "android.location.Location", "android.provider.Settings", "android.util.Log", "java.util.List", "fuse.geolocation.UpdateListener", "android.os.Looper", "android.content.Context", "com.uno.StringArray")]
+
```
Comment (mortend) [inline Source/Fuse.GeoLocation/Android/AndroidLocationProvider.uno]: Code-style nit: Drop superfluous newline.

**Comment by mortend** [comment]
Should we rebase this on latest master to lose the merge-commits?

**Comment by AndrewEQ** [comment]
Sure thing, I'm on it...

**Pull request merged**